src/Documentation/fpga/xrt.rst

```diff
@@ -596,6 +596,72 @@ The driver object model looks like the following::
     | leaf |...| leaf |     | leaf |...| leaf |
     +------+   +------+     +------+   +------+
 
+One example on Alveo U50 before xclbin download looks like the following::
+
+                                +-----------+
+                                |   xmgmt   |
+                                +-----+-----+
+                                      |
+            +-------------------------+--------------------+
+            |                         |                    |
+            v                         v                    v
+       +--------+                +--------+            +--------+
+       | group0 |                | group1 |            | group2 |
+       +----+---+                +----+---+            +---+----+
+            |                         |                    |
+            |                         |                    |
+      +-----+-----+             +-----+---+          +-----+----+
+      |           |             |         |          |          |
+      v           v             v         v          v          v
+ +------------+  +------+    +------+ +------+    +-----+ +-----------+
+ | xmgmt_main |  | VSEC |    | GPIO | | QSPI |    | CMC | | AXI-GATE0 |
+ +------------+  +------+    +------+ +------+    +-----+ +-----------+
+                             +---------+          +------+ +-------+
+                             | MAILBOX |          | ICAP | | CALIB |
+                             +---------+          +------+ +-------+
+                                                  +-----------+
+                                                  | AXI-GATE1 |
+                                                  +-----------+
```

Comment:
Looks good. Wondering if we should draw arrow from group1 to MAILBOX and group2 to ICAP, CALIB, AXI-GATE1?

Reply:
fixed